proto/internal.proto

```diff
@@ -453,5 +453,7 @@ message RaftSnapshotData {
     optional bytes key = 1;
     optional bytes value = 2;
   }
-  repeated KeyValue KV = 1 [(gogoproto.customname) = "KV"];
+  // The latest RangeDescriptor
+  optional RangeDescriptor range_descriptor = 1 [(gogoproto.nullable) = false];
+  repeated KeyValue KV = 2 [(gogoproto.customname) = "KV"];
```

Comment:
But it's more work to break compatibility than to preserve it :). There's no benefit to renumbering things to match textual order. I think there's enough client development now that we should preserve tag numbers in client-facing protos (when it's easy to do so; if we have a reason to make incompatible changes we still can). But in internal.proto it doesn't matter either way.